docs/UsersGuide/source/BuildingRunningTesting/VXCases.rst

```diff
@@ -141,9 +133,15 @@ Then, edit the configuration file (``config.yaml``) to include the variables and
       # Add EXTRN_MDL_SOURCE_BASEDIR_LBCS variable to config.yaml
       EXTRN_MDL_SOURCE_BASEDIR_LBCS: /path/to/Indy-Severe-Weather/input_model_data/FV3GFS/grib2/2019061500
       USE_USER_STAGED_EXTRN_FILES: true
-   task_run_fcst:
-      WTIME_RUN_FCST: 05:00:00
-      PREDEF_GRID_NAME: SUBCONUS_Ind_3km
+   verification:
+     VX_FCST_MODEL_NAME: FV3_GFS_v16_SUBCONUS_3km
+   rocoto:
+     tasks:
+       metatask_run_ensemble:
+         task_run_fcst_mem#mem#:
+           walltime: 02:00:00
+       taskgroups: '{{ ["parm/wflow/prep.yaml", "parm/wflow/coldstart.yaml", "parm/wflow/post.yaml", "parm/wflow/verify_pre.yaml", "parm/wflow/verify_det.yaml"]|include }}'
+      
```

Comment:
@EdwardSnyder-NOAA I will need your feedback or a copy of your config file to ensure that the changes to this config.yaml excerpt are correct and complete.

Reply:
Three changes to the config file:

-  Remove the `NOHRSC_OBS_DIR` variable because we don't use these obs for this case.
- Change the physics suite variable to RRFS beta1 i.e. `CCPP_PHYS_SUITE: FV3_RRFS_v1beta`. The reason being is that this physics suite better matches the grid we are using.
- Change the verification name i.e. `VX_FCST_MODEL_NAME: FV3_RRFS_v1beta_SUBCONUS_Ind_3km`. Making this change means that the vx files mentioned throughout this document needs to change as well. See my other comment.

Here is the config file I used to run the experiment:
```
metadata:
  description: >-
    Sample community config
user:
  RUN_ENVIR: community
  MACHINE: hera
  ACCOUNT: epic
platform:
  CCPA_OBS_DIR: "/scratch1/NCEPDEV/nems/Edward.Snyder/indy-srw-sample-vx-2p2/Indy-Severe-Weather/obs_data/ccpa/proc"
  MRMS_OBS_DIR: "/scratch1/NCEPDEV/nems/Edward.Snyder/indy-srw-sample-vx-2p2/Indy-Severe-Weather/obs_data/mrms/proc"
  NDAS_OBS_DIR: "/scratch1/NCEPDEV/nems/Edward.Snyder/indy-srw-sample-vx-2p2/Indy-Severe-Weather/obs_data/ndas/proc"
workflow:
  USE_CRON_TO_RELAUNCH: false
  EXPT_SUBDIR: test_community
  CCPP_PHYS_SUITE: FV3_RRFS_v1beta
  PREDEF_GRID_NAME: SUBCONUS_Ind_3km
  DATE_FIRST_CYCL: '2019061500'
  DATE_LAST_CYCL: '2019061500'
  FCST_LEN_HRS: 60
  PREEXISTING_DIR_METHOD: rename
  VERBOSE: true
  COMPILER: intel
task_get_extrn_ics:
  EXTRN_MDL_NAME_ICS: FV3GFS
  FV3GFS_FILE_FMT_ICS: grib2
  EXTRN_MDL_SOURCE_BASEDIR_ICS: /scratch1/NCEPDEV/nems/Edward.Snyder/indy-srw-sample-vx-2p2/Indy-Severe-Weather/input_model_data/FV3GFS/grib2/2019061500
  USE_USER_STAGED_EXTRN_FILES: true
task_get_extrn_lbcs:
  EXTRN_MDL_NAME_LBCS: FV3GFS
  LBC_SPEC_INTVL_HRS: 6
  FV3GFS_FILE_FMT_LBCS: grib2
  EXTRN_MDL_SOURCE_BASEDIR_LBCS: /scratch1/NCEPDEV/nems/Edward.Snyder/indy-srw-sample-vx-2p2/Indy-Severe-Weather/input_model_data/FV3GFS/grib2/2019061500
  USE_USER_STAGED_EXTRN_FILES: true
task_run_fcst:
  QUILTING: true
task_plot_allvars:
  COMOUT_REF: ""
global:
  DO_ENSEMBLE: false
  NUM_ENS_MEMBERS: 2
verification:
  VX_FCST_MODEL_NAME: FV3_RRFS_v1beta_SUBCONUS_Ind_3km
rocoto:
  tasks:
    metatask_run_ensemble:
      task_run_fcst_mem#mem#:
        walltime: 02:00:00
    taskgroups: '{{ ["parm/wflow/prep.yaml", "parm/wflow/coldstart.yaml", "parm/wflow/post.yaml", "parm/wflow/verify_pre.yaml", "parm/wflow/verify_det.yaml"]|include }}'
```